### PR TITLE
TUTORIAL Updated by @emmanuelJet

### PR DESCRIPTION
**Pull request by emmanuelJet**
TUTORIAL: How to Register and Update a Name with the SDK in Javascript updated by @emmanuelJet. @thepiwo, @alekseytsekov, and @Pegahbit kindly review.

AE Address: ak_rLoCtHE3NK9dKyCNonJFYWkEEfeAsDUWa887GsCKqV1rhSuT6

### CODE REVIEW

**Comment by emmanuelJet** [comment]
Confirmation image added to the new commit

File: naming-workflow-in-javascript-sdk.md [hunk 1]
```diff
@@ -155,13 +185,11 @@ const main = async (name) => {
   console.log(update);
 };
 
-main('YOUR_NAME_TO_REGISTER.test');
+main('emmanueljet.test');
```
Comment (alekseytsekov) [inline naming-workflow-in-javascript-sdk.md]: you should await `main`

**Comment by emmanuelJet** [comment]
TUTORIAL: TUTORIAL: How to Get Started With Unit Testing Using forgae updated by @emmanuelJet on last commit. @thepiwo, @alekseytsekov, and @Pegahbit kindly review.

AE Address: ak_rLoCtHE3NK9dKyCNonJFYWkEEfeAsDUWa887GsCKqV1rhSuT6

File: naming-workflow-in-javascript-sdk.md (hunk 1 above)
Comment (thepiwo) [reply naming-workflow-in-javascript-sdk.md]: main can't be awaited there

Comment (thepiwo) [reply naming-workflow-in-javascript-sdk.md]: @emmanuelJet can you revert to `YOUR_NAME_TO_REGISTER.test`

Comment (emmanuelJet) [reply naming-workflow-in-javascript-sdk.md]: Will do that now

Comment (emmanuelJet) [reply naming-workflow-in-javascript-sdk.md]: @thepiwo `emmanueljet.test` has being reverted to `YOUR_NAME_TO_REGISTER.test`

**Comment by thepiwo** [comment]
Hello, can you resolve the conflicts, so that we can merge

**Comment by emmanuelJet** [comment]
> Hello, can you resolve the conflicts, so that we can merge

I had to create another PR #48 for the merging

**Pull request closed without merging**